src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Link from 'app/components/links/link';
+import {PanelItem} from 'app/components/panels';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import space from 'app/styles/space';
+import {Organization, SentryApp} from 'app/types';
+import {t} from 'app/locale';
+
+import IntegrationStatus from './integrationStatus';
+
+type Props = {
+  organization: Organization;
+  type: 'plugin' | 'firstParty' | 'sentryApp';
+  slug: string;
+  displayName: string;
+  status: 'Installed' | 'Not Installed' | 'Pending';
+  publishStatus: 'unpublished' | 'published' | 'internal';
+  configurations: number;
+};
+
+const urlMap = {
+  plugin: 'plugins',
+  firstParty: 'integrations',
+  sentryApp: 'sentry-apps',
+};
+
+const IntegrationRow = (props: Props) => {
+  const {
+    organization,
+    type,
+    slug,
+    displayName,
+    status,
+    publishStatus,
+    configurations,
+  } = props;
+
+  const baseUrl = `/settings/${organization.slug}/${urlMap[type]}/${slug}/`;
+
+  const renderDetails = () => {
+    if (type === 'sentryApp') {
+      return publishStatus !== 'published' && <PublishStatus status={publishStatus} />;
+    }
+    return configurations > 0 ? (
+      <StyledLink to={`${baseUrl}?tab=configurations`}>{`${configurations} Configuration${
+        configurations > 1 ? 's' : ''
+      }`}</StyledLink>
+    ) : null;
+  };
+
+  return (
+    <PanelItem p={0} flexDirection="column" data-test-id={slug}>
+      <FlexContainer>
+        <PluginIcon size={36} pluginId={slug} />
+        <Container>
+          <IntegrationName to={baseUrl}>{displayName}</IntegrationName>
```

Comment:
@scefali I think I mentionned this in one of your PR's but should this be wrapped by `t()` I don't remember what Evan replied with

Reply:
@leeandher This is a tough one. I don't think we should actually translate the names of applications. We might translate the word `Legacy`, however.